### PR TITLE
Searchlight service

### PR DESCRIPTION
**Pull request by mateusz-blaszkowski**
Signed-off-by: Mateusz Blaszkowski <mateusz.blaszkowski@intel.com>

### CODE REVIEW

File: stackanetes-services/searchlight/searchlight-init-indexes.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+type: generic/job
+name: searchlight-init-indexes
+command: bash /tmp/searchlight-init-indexes.sh
+label: openstack-control-plane
+image: searchlight
+dependencies:
+  service:
+  - elasticsearch-search
+  - ironic-api
```
Comment (DTadrzak) [inline stackanetes-services/searchlight/searchlight-init-indexes.yml]: Currently we don't support ironic.

File: stackanetes-services/searchlight/searchlight-init.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+type: generic/job
+name: searchlight-init
+command: bash /tmp/searchlight-init.sh
+label: openstack-control-plane
+image: centos-binary-kolla-toolbox
```
Comment (DTadrzak) [inline stackanetes-services/searchlight/searchlight-init.yml]: please remove that redundant prefix "centos-binary" from image. Prefix is configurable. If u use kolla-toolbox you don't even need to specify image name it's default value for jobs.

File: config/searchlight/templates/searchlight.conf.j2 [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+[api]
+bind_port = {{ searchlight_api_port }}
+workers = 4
+
+[elasticsearch]
+hosts = {{ elasticsearch_host }}:{{ elasticsearch_port }}
+
+[listener]
+workers = 4
```
Comment (lleszczu) [inline config/searchlight/templates/searchlight.conf.j2]: move value to template

**Comment by DTadrzak** [comment]
Please add information to README that horizon docker image required plugin to use searchlight.

**Pull request merged**